go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	cloud.google.com/go/iam v1.1.1 // indirect
 	cloud.google.com/go/storage v1.30.1 // indirect
 	dario.cat/mergo v1.0.0 // indirect
+	github.com/IBM/cloud-databases-go-sdk v0.5.0
```

Comment:
Did you manually add this entry here? If so, it goes in the `require` section above with all the other IBM libraries, the lower require section is usually reserved for `gomod` indirect additions